frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form/ocs-storage-class-form.tsx

```diff
@@ -543,17 +374,134 @@ export const StorageClassEncryptionKMSID: React.FC<ProvisionerProps> = ({
     namespace: CEPH_STORAGE_NAMESPACE,
     name: KMSConfigMapCSIName,
   };
+  const [csiConfigMap, csiConfigMapLoaded, csiConfigMapLoadError] = useK8sWatchResource<
+    ConfigMapKind
+  >(csiCMWatchResource);
```

Comment:
Please add useDeepCompareMemorize after watch, otherwise, your useEffect will be executed again and again for object as its dependency

Reply:
ack...